docs/api/XSK_BUFFER_DESCRIPTOR.md

```diff
@@ -0,0 +1,26 @@
+# XSK_BUFFER_DESCRIPTOR structure
+
+A structure for an AF_XDP data buffer.
+
+## Syntax
+
+```C
+#define XSK_BUFFER_DESCRIPTOR_ADDR_OFFSET_MAX 65535ull
+#define XSK_BUFFER_DESCRIPTOR_ADDR_OFFSET_SHIFT 48
+#define XSK_BUFFER_DESCRIPTOR_ADDR_OFFSET_MASK \
+    (XSK_BUFFER_DESCRIPTOR_ADDR_OFFSET_MAX << XSK_BUFFER_DESCRIPTOR_ADDR_OFFSET_SHIFT)
+
+typedef struct _XSK_BUFFER_DESCRIPTOR {
+    // Bits 0:47 encode the address of the chunk, relative to UMEM start address.
+    // Bits 48:63 encode the packet offset within the chunk.
+    UINT64 Address;
```

Comment:
I'm curious why you don't specify this as a bitfield then?

Reply:
IDK, I did not write this. I think there may have been a concern that other compilers may use different bitfield layouts, but in practice any compiler used on Windows code must use Windows-compatible bitfields. So this might be a good thing to go and fix now.

Reply:
Opened #345